rosetta/rosetta/projects/pax/README.md

```diff
@@ -4,6 +4,9 @@
 Any `paxml/*` or `praxis/*` relative directory/file can be found in [google/paxml](https://github.com/google/paxml/tree/main) or [google/praxis](https://github.com/google/praxis/tree/main), respectively, but to
 view the most up-to-date version of that directory/file with any GPU-specific patches, please see [Inspecting the source code](#inspecting-the-source-code).
 
+## Hardware Specifications
+Convergence and performance has been validated on NVIDIA DGX A100 (8x A100 80G) nodes. We provide both singlenode and multinode pre-training support. If running on a machine with less than 80G memory, some of the default configurations may run out of memory; if you run out of memory and have more GPUs available, increase your GPU count and decrease your batch size per GPU.
```

Comment:
It would be great if we can link to some artifacts from the claimed successful run.

Reply:
Convergence logs are linked in the [configs section](https://github.com/NVIDIA/JAX-Toolbox/blob/98278cc443a03f4089a505f83b6f29ec43097fc2/rosetta/rosetta/projects/pax/README.md#configs). Is this sufficient, or do you think we should upload additional aritfacts?

Reply:
How about just say something like
> Convergence and performance has been validated on NVIDIA DGX A100 (8x A100 80G) nodes. For details, see the [Configs](#configs) section below.
?

Reply:
Here, we use Markdown anchors to link to the Config section on the same page.

Reply:
Sounds good, I made that change